### ISSUE TITLE
Missing <label> - Content at the Search Bar

### ISSUE THREAD

**Issue by brunobeeee** [denitcoder/wp-theme-zentile#17]
The main search bar at the top of the homepage has no content inside the <label> structure. Therefore screen readers cannot tell the user what this input form is for.

My best idea is to add some content to the to the <label> but make it "invisible" to the user by some CSS magic. This doesn't change the design of the homepage but it allows screen readers to work properly.
